src/operator/contrib/roi_align.cc

```diff
@@ -156,7 +157,10 @@ void ROIAlignForward(
   int n_rois = nthreads / channels / pooled_width / pooled_height;
   // (n, c, ph, pw) is an element in the pooled output
   // can be parallelized using omp
-  for (int n = 0; n < n_rois; n++) {
+  int n;
```

Comment:
Is it possible to move this definition into the for-loop?

Reply:
Ok, the private() clause is removed.